### PR TITLE
Updated action list in token, NFTs and activity view

### PR DESCRIPTION
**Pull request by NidhiKJha**
This PR is to :
1. Update the action list in Tokens, Nfts and Activity to include MetaMask support
2. For the NFTs tab, if the user has autodetect enabled, then display Refresh List otherwise display enable autodetect
3. For activity screen, support is to be in center
4. Spacing between each action button should be 8px

* Fixes [#711](https://github.com/MetaMask/MetaMask-planning/issues/711)

## Screencaps

https://github.com/MetaMask/metamask-extension/assets/39872794/fa232f76-fda0-49c2-99df-6cad3f03c075

## Manual Testing Steps

- Go to NFT tabs, look for the updated UI for the actions button
- Check support link is at the left for all the three tabs

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by darkwing — APPROVED** [review]

**Review by darkwing — CHANGES_REQUESTED** [review]
Functionally looks good but requires jest updates.

**Review by albertolive — DISMISSED** [review]

**Comment by darkwing** [comment]
Looks great!  The only thing I'm noticing is that the space between the links on the Tokens and NFTs tab looks different:

SCR-20230622-jwnq

SCR-20230622-jwyt

**Comment by NidhiKJha** [comment]
> Looks great! The only thing I'm noticing is that the space between the links on the Tokens and NFTs tab looks different:
> 
Fixed here https://github.com/MetaMask/metamask-extension/pull/19702/commits/e62431d7b36a14e6dad470725c96d3c9df5800b8

**Review by albertolive — DISMISSED** [review]

**Review by vthomas13 — DISMISSED** [review]

File: ui/components/app/nfts-tab/nfts-tab.js [hunk 1]
```diff
@@ -60,89 +55,53 @@ export default function NftsTab({ onAddNFT }) {
...
+                </ButtonLink>
+              ) : (
+                <ButtonLink
+                  size={Size.MD}
+                  startIconName={IconName.Refresh}
+                  data-testid="refresh-list-button"
+                  onClick={onRefresh}
+                >
+                  {t('refreshList')}
+                </ButtonLink>
```
Comment (darkwing) [inline ui/components/app/nfts-tab/nfts-tab.js:100]: I think we can incorporate this as well!

https://github.com/MetaMask/metamask-extension/pull/19743

**Review by darkwing — CHANGES_REQUESTED** [review]

File: ui/components/app/nfts-tab/nfts-tab.js (hunk 1 above)
Comment (NidhiKJha) [reply ui/components/app/nfts-tab/nfts-tab.js:100]: I think this already exists here for NFTs

https://github.com/MetaMask/metamask-extension/assets/39872794/105032f0-f274-4aa3-8bf2-b08a6b16db1c

**Review by darkwing — APPROVED** [review]

**Review by darkwing — CHANGES_REQUESTED** [review]
Hmm, actually, shouldn't there be a "no nfts" message here?
SCR-20230626-lsdf

**Comment by NidhiKJha** [comment]
> Hmm, actually, shouldn't there be a "no nfts" message here? 

Nice shout, Updated! here https://github.com/MetaMask/metamask-extension/pull/19702/commits/2496b2398d7e6005d2edf577335d7deedfcf1dd7

**Review by darkwing — APPROVED** [review]

**Review by albertolive — APPROVED** [review]

**Pull request merged**